homeassistant/components/tado/config_flow.py

```diff
@@ -66,7 +66,7 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    async def async_step_user(self, user_input=None):
+    async def async_step_user(self, user_input=None) -> FlowResult:
```

Comment:
```suggestion
    async def async_step_user(self, user_input: dict[str, Any] | None=None) -> FlowResult:
```
Might need to import Any if that's not already done